### PR TITLE
Recommendations UX

### PR DESCRIPTION
**Pull request by michal-zielonka**
## Description

I am Adjusting Recommendations to the SWO Design System based on Figma designs. I am also adding minor changes to related components (SideDrawer, Chips, MultiButton, Icon, and more). 

Main view
Screenshot 99

Hover states for top cards
Screenshot 100

Table view
Screenshot 101

Select combo box with active items
Screenshot 102

Side Drawer - flat title color
Screenshot 103

Drawer - second option with a gradient title (the one that actually is in the code) - feedback is appreciated.
Screenshot 104

## Related issue number

MPT-4864

## Special notes

N/A

## Checklist

* [ ] The pull request title is a good summary of the changes
* [ ] Unit tests for the changes exist
* [ ] New and existing unit tests pass locally

### CODE REVIEW

File: ngui/ui/src/components/SummaryCard/SummaryCard.tsx [hunk 1]
```diff
@@ -24,8 +24,8 @@ const CardLayout = forwardRef(({ children, color, clickable, onClick, cardTestId
         <Box>
           {children}
           {clickable && (
-            <Box position="absolute" bottom={0} right={0}>
-              <TouchAppIcon color="info" sx={{ opacity: 0.5 }} />
+            <Box position="absolute" bottom={"16px"} right={"8px"}>
```
Comment (dwolosz) [inline ngui/ui/src/components/SummaryCard/SummaryCard.tsx:27]: Can we use MPT_SPACING_1 and _2 ?

File: ngui/ui/src/hooks/useRecommendationServices.ts [hunk 2]
```diff
@@ -105,10 +105,10 @@ export const useRecommendationServices = () => {
     [ALL_SERVICES]: {
       name: ALL_SERVICES
     },
-    ...ALIBABA_SERVICES,
-    ...AWS_SERVICES,
+    // ...ALIBABA_SERVICES,
```
Comment (dwolosz) [inline ngui/ui/src/hooks/useRecommendationServices.ts:108]:
```suggestion
    // MPT_TODO: disabled to math BDR requirements
    // ...ALIBABA_SERVICES,
```

File: ngui/ui/src/theme.ts [hunk 3]
```diff
@@ -463,6 +509,12 @@ const getThemeConfig = (settings = {}) => {
               padding: MPT_SPACING_3,
               borderRadius: MPT_SPACING_1
             }
+          },
+          ".MuiBox-WhiteCard": {
+            boxShadow: MPT_BOX_SHADOW,
+            background: "#FFFFFF",
```
Comment (dwolosz) [inline ngui/ui/src/theme.ts:515]:
```suggestion
            background: MPT_BRAND_WHITE,
```

plus add import of course

Comment (michal-zielonka) [reply ngui/ui/src/theme.ts]: Fixed

**Review by dwolosz — APPROVED** [review]

**Pull request merged**